bootstrap/001_schemas_and_roles.sql

```diff
@@ -35,3 +35,5 @@ GRANT USAGE ON SCHEMA ADMIN TO APPLICATION ROLE ADMIN;
 GRANT USAGE ON SCHEMA ADMIN TO APPLICATION ROLE SUNDECK_SERVICE_ROLE;
 
 create view catalog.config if not exists as select * from internal.config;
+
+GRANT CREATE WAREHOUSE ON ACCOUNT TO APPLICATION ROLE ADMIN;
```

Comment:
also, ALTER/DROP ?

Reply:
I think we don't need it 

https://docs.snowflake.com/en/user-guide/security-access-control-privileges
`OWNERSHIP is a special privilege on an object that is automatically granted to the role that created the object`

Reply:
I verfied in Sf, with only Create Warehouse privliege, I could delete, created warehouse

Reply:
ok - also try changing the tag of warehouse, adding it to a resource monitor. see [here](https://docs.snowflake.com/en/user-guide/security-access-control-privileges#virtual-warehouse-privileges)

Reply:
did you try this in a full deployment? I don't think this will work (or is required) if the native app has the correct permissions

Reply:
yes, this won't be needed.
I realized that this won't work with failed PR test. I am fixing it to ask permission from user as discussed on slack

Reply:
fixed warehouse permission

Reply:
For tag -- We can't use tag in standard edition
Resource monitor -- only account admin can create it